sdk-tests/deploy/local-test-vault.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  hashicorp_vault:
+    image: vault
+    ports:
+      - "8200:8200"
+    cap_add:
+      - IPC_LOCK
+    environment:
+      VAULT_DEV_ROOT_TOKEN_ID: "myroot"
```

Comment:
Seems duplicate but this one is for tests. Sample might use another vault in the future, but we keep this one for tests. They are the same today.